src/theme/components/helperErrorText.ts

```diff
@@ -1,20 +1,31 @@
-interface HelperErrorTextBaseStyle {
+import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
+import { defineStyle, StyleFunctionProps } from "@chakra-ui/system";
+
+interface HelperErrorTextBaseStyle extends StyleFunctionProps {
```

Comment:
Just FYI, I ran into an issue where this caused a problem:

<img width="722" alt="Screen Shot 2023-12-06 at 4 55 17 PM" src="https://github.com/NYPL/nypl-design-system/assets/1280564/ee203676-9c0d-4e7f-bcfa-b5c0c65cd641">

It's not a problem for this component or interface but if you run into the same problem, it can be fixed by making it a partial `Partial<StyleFunctionProps>`. We do need to extend the interface but we don't need to make all the properties required.

Reply:
To keep it consistent I make the change here too. It might be better in the long run.

Reply:
We talked about this offline but for history: we couldn't figure out why Chakra is being inconsistent and why not using `Partial<StyleFunctionProps>` in this case breaks. So for this case (it might be a case-by-case basis), it's okay to not use `Partial`.